srcpkgs/librespeed-cli/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'librespeed-cli'
+pkgname=librespeed-cli
+version=1.0.10
+revision=1
+build_style=go
+go_import_path=github.com/librespeed/speedtest-cli
+short_desc="LibreSpeed command line tool"
+maintainer="Massimo Manzato <massimo.manzato@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/librespeed/speedtest-cli"
+distfiles="https://github.com/librespeed/speedtest-cli/archive/refs/tags/v${version}.tar.gz"
+checksum=904ec84f41714d5a7ab778534d332219cd254fdd0f97cc33ebb3540d31fb802c
+
+post_install() { # Rename the executable due to conflic with PKG speedtest-cli
```

Comment:
nitpick: comment on a different line than the function definition

Reply:
done

Reply:
> nitpick: comment on a different line than the function definition

Hi @classabbyamp,

anythings else needed?  
how the merge into master branch work ? where can I find useful information regarding void-packages workflow? 

I'm interested into try to build a kernel template for OrangePI zero 2 plus, and orangePi 3LTS (like fro DROID ), how I can ask for help on this ? 

many thanks 
Max